### DISCUSSION TITLE
Support for non-string values for Select components

### DISCUSSION THREAD

**Discussion by yousefelgoharyx** [mantinedev/mantine#1804] [Feature requests]
Currently select component only accepts strings a value. Sometimes it's just an ID.
Calling toString method everywhere just doesn't seem right

**Comment by yousefelgoharyx** [comment]
Any news? Having only string values in the select component is a huge pain

**Reply by rtivital** [reply]
I'm not planning to add support for non-string values any time soon – it will make Select component a lot harder to maintain.
